AppRest-melvs/container/src/main/java/com/practice/MelvsBoot/BootShop/controller/PageController.java

```diff
@@ -0,0 +1,79 @@
+package com.practice.MelvsBoot.BootShop.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.practice.MelvsBoot.BootShop.model.Accessories;
+import com.practice.MelvsBoot.BootShop.model.MyBike;
+import com.practice.MelvsBoot.BootShop.repository.MyAccessoriesService;
+import com.practice.MelvsBoot.BootShop.repository.MyBikeService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Collection;
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/page")
+public class PageController {
+
+    MyBikeService mybike = new MyBikeService();
+    @RequestMapping(method = RequestMethod.GET, value = "/bikes")
+    @ResponseStatus(HttpStatus.OK)
+    public List<MyBike> getAllList() {
+        return mybike.getBikeList();
+    }
+
+    private Collection<Accessories> allaccessories;
+    public PageController(){
+        MyAccessoriesService acrepo = new MyAccessoriesService();
+        this.allaccessories = acrepo.getAcc();
+    }
+    @RequestMapping(method = RequestMethod.GET, value = "/accessories")
+    @ResponseStatus(HttpStatus.OK)
+    public String allaccesories(){
+        Collection<Accessories> response = this.allaccessories;
+        String responseString = this.convertResponseToString(response);
+        System.out.println(responseString);
+        return responseString;
+    }
+
```

Comment:
Would be great if you could use ResponseEntity here, for you to be acquainted with both ResponseEntity and ObjectMapper usage, but no need to implement it right now. In the future, when applicable, I'd recommend using ResponseEntity.